Services/OpenIdConnect/classes/class.ilAuthProviderOpenIdConnect.php

```diff
@@ -23,11 +23,18 @@
 class ilAuthProviderOpenIdConnect extends ilAuthProvider
 {
     private ilOpenIdConnectSettings $settings;
+    /** @var mixed | null | object */
+    private $body;
```

Comment:
@pascalseeland IMO this is always an array `$DIC->http()->request()->getParsedBody();`

Reply:
According to PSR7 ServerRequestInterface->getParsedBody(), an Implementation can return any of those parameters.

Reply:
Yes, you are right, according to the PSR standard this could be of different types. Maybe it is wrong that we treat this as an array in our code base.

The type is `null|array|object`, see `\Psr\Http\Message\ServerRequestInterface::getParsedBody`, you used `mixed | null | object`.